.travis.yml

```diff
@@ -10,7 +10,11 @@ env:
 install: true
 
 script:
-  - go get -u golang.org/x/lint/golint
+  - set -e
+  - fmt=$(gofmt -l -s .)
+  - test -z $fmt || (echo "please run gofmt" ; echo $fmt ; exit 1)
+  - go get golang.org/x/lint/golint
   - golint -set_exit_status $(go list ./...)
-  - go test -test.v -test.vet=all -test.count 1 ./...
-  - go test -test.v -test.race ./...
+  - go generate ./...
+  - git diff --quiet || (echo 'generated go file are not up to date' ; exit 1)
+  - go test -race -vet=all ./...
```

Comment:
no verbose mode anymore?

Reply:
verbose adds lots of output for passing tests, which makes finding failures hard. for debugging you can run it locally however you like, but i wouldn't use -v by default.